lib/interactive.dart

```diff
@@ -87,10 +88,6 @@ class Interactive {
 
   init() async {
     _container.nodes.clear();
-    var placeholder = html.ParagraphElement()
-      ..classes = [ANIMATE_BLINK_CSS_CLASS]
-      ..appendText(CHART_LOADING_PLACEHOLDER_TEXT);
```

Comment:
Are `CHART_LOADING_PLACEHOLDER_TEXT` and `CHART_CLEAR_PLACEHOLDER_TEXT` (below) still being used somewhere or have they been replaced by the 'Loading data...' in html?

Reply:
Ah yes good catch, they both are deprecated. Will remove it in the next PR.